framework/wazuh/cdb_list.py

```diff
@@ -61,11 +61,11 @@ def _iterate_lists(absolute_path, only_names=False):
             and ('.cdb' not in name)  \
             and ('~' not in name) \
             and not pattern.search(name):
-            items = get_list_from_file(new_relative_path)
             if only_names:
                 relative_path = _get_relative_path(absolute_path)
                 output.append({'path': relative_path, 'name': name})
             else:
+                items = get_list_from_file(new_relative_path)
```

Comment:
The docstring of this function is obsolete. Could you please update it? Thanks!

Reply:
done!